### ISSUE TITLE
v1.25.0 C# extension not working - Unity 2021 & 2020

### ISSUE THREAD

**Issue by ahmethanfurkan** [OmniSharp/omnisharp-vscode#5252]
Issue Type: <b>Bug</b>

Using Unity 2021 & 2020, C# extension old version (ex: v1.24.4) working but v1.25.0 codes not working (IntelliSense), no errors.

Extension version: 1.24.4
VS Code version: Code 1.67.2 (c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5, 2022-05-17T18:15:52.058Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No

<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392:30443607
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
vscscmwlcmt:30465135
cppdebug:30492333
vsclangdc:30486549

```

</details>

**Comment by filipw** [comment]
Please see the announcement https://github.com/OmniSharp/omnisharp-vscode#the-c-extension-no-longer-ships-with-an-included-mono--msbuild-tools

and the related issue https://github.com/OmniSharp/omnisharp-vscode/issues/5120

TL;DR

 - ensure .NET SDK 6 is installed
 - for .NET Core (and newer) development this should be enough
 - for Unity / .NET Framework development, set "omnisharp.useModernNet" : false and ensure global MSBuild is installed on your machine - either using VS 2022 Build Tools (Windows) or using latest [Mono Preview](https://www.mono-project.com/download/preview/) (*nix)
